packages/flutter/lib/src/material/i18n/material_en.arb

```diff
@@ -47,6 +47,12 @@
     "type": "text"
   },
 
+  "aboutListTileTitle": "About $applicationName",
+  "@aboutListTileTitle": {
+    "description": "The default title for AboutListTile",
+    "type": "text"
+  },
```

Comment:
do we document the magic substitution strings?

Reply:
There is some info about resource ids and the details about how the *.arb files are mapped to a per locale table of resourceId to value tables in dev/tools/gen_localizations.dart.

I need to add an i18n/README.md that explains what's going on more clearly.

Reply:
https://github.com/flutter/flutter/issues/12113